torch/testing/_internal/common_distributed.py

```diff
@@ -1502,24 +1500,29 @@ def _run(
 
 class MultiProcContinousTest(TestCase):
     # Class variables:
+    MAIN_PROCESS_RANK = -1
     # number of test processes
-    world_size: int = 2
+    world_size: int = -2  # unset state
```

Comment:
why are we changing this to -2?

Reply:
will be determined at run time by device count.